aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1 +1,361 @@
+
+**Image goes here**
+
+Problems:
+- If possible, give the user a way to directly perform a mentioned action.
+
+**Image goes here**
+
+## Miscellany
+
+### Quoting
+
+In messaging that contains user-specified content, use double-quotes around this content if it is not rendered as a hyperlink. If user-content is rendered as a hyperlink, the hyperlink styling is sufficient to offset the content from other text in the message.
+
+Why is this important? User-entered names can contain spaces, and if they are not quoted or rendered as hyperlinks, they can hinder comprehension of a message.  
+
+Here is a correct example: *the uplink set "{0}" has a problem*. Compare this to an incorrect example: *the uplink set {0} has a problem*. If the uplink set is named "deleted", this will read as *the uplink set deleted has a problem*.
+
+### Referring to support
+
+- Do not tell the user to contact their administrator. The user is likely the administrator.
+- Do not tell the user to contact HPE. A 3rd party may be providing support to the customer.
+- Use something like: *Contact your authorized support representative.*
+- Do not overuse this phrase. If the resolution to a problem is clear, just give the resolution and do not add that the user should contact support.
+- Contacting support costs everypone time and money, so do not turn every issue into a potential support issue. It makes our product seem unnecessarily fragile and can result in unnecessary support calls.
+- Do use this phrase for internal errors where there's nothing else the user can do (i.e. when it is the only resolution).
+
+### Localization
+
+A best practice for developers is to add comments to messages in the properties files or message catalogs. Reviewers and translators (and ultimately, customers) will thank you.
+
+- Document all parameter substitution values. When the value is a resource, be specific. Is it a name? Is it a URI? Is it raw JSON that needs to be encapsulated in curly braces? Or is it pre-formatted JSON that includes curly braces?
+- For errors and alerts, document the severity associated with the message.
+- Document the type of message (alert, task, etc).
+- If the message doesn't document (for the user) full details of how this problem occurs or what to do about it, add the details in comments.
+- Some context around the task being performed or the resource category involved is useful.
+- Is the message an API-only message (does not appear in the UI)? If yes, document it as such.
+
+### Scale
+
+Consider singular and plural use cases and craft messages for each.
+
+If a sentence contains a list, do not allow the length of the list to be unbounded. Instead, consider a construct like this: *The following policies have not been implemented: Policy A, Policy B, Policy C, and 5 others.*
```

Comment:
Is there any guidance on how many items to show before bounding? > 3 maybe?

Reply:
>3 is what we have used in the past.